native/src/components/MalteHelpFormOffer.tsx

```diff
@@ -2,6 +2,7 @@ import React, { ReactElement } from 'react'
 import { useForm } from 'react-hook-form'
 import { useTranslation } from 'react-i18next'
 import { View } from 'react-native'
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view'
```

Comment:
:thought_balloon: Last update of the library was 3 years ago, perhaps we should also switch libraries at some point.

Reply:
Good idea, I opened #2743 for it :)